elements/pfe-jump-links/src/pfe-jump-links-nav.scss

```
@@ -23,7 +30,7 @@ $variables: (
   font-weight: 400; // remove this once font-weight-normal is 400
   display: block;
   position: sticky;
-  top: calc(#{pfe-var(container-spacer)} * 4);
+  top:  #{pfe-local($cssvar: offset, $region: nav)};
```

```suggestion
  top:  pfe-local(offset, $region: nav);
```

No need to interpolate here, I also think it might be easier to read this way (without the $cssvar).  What do you think?